common/views/components/MetaUnit/MetaUnit.js

```diff
@@ -38,16 +38,7 @@ const Paragraphs = ({ text }) => {
   return (
     text.length > 0 &&
     text.map((para, i) => {
-      return (
-        <p
-          key={i}
-          className={`${font({ s: 'HNL5', m: 'HNL4' })} ${spacing(
-            { s: 2 },
-            { margin: ['bottom'] }
-          )}`}
-          dangerouslySetInnerHTML={{ __html: para }}
-        />
-      );
+      return <p key={i} dangerouslySetInnerHTML={{ __html: para }} />;
```

Comment:
It would be good to not be using `dangerouslySetInnterHTML` here, bikeshedding though.